text/0000-generic-int.md

```diff
@@ -0,0 +1,552 @@
+- Feature Name: generic_int
+- Start Date: 2018-05-15
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Adds the builtin types `uint<N>` and `int<N>`, allowing integers with an
```

Comment:
Perhaps the types should be called `UInt<N>` and `Int<N>` since that is more conventional these days; however, all the primitive types are lower cased so perhaps not... I'm torn.

Reply:
The weird thing is that they are generic primitives, which I've never seen in a language before...

Perhaps we should do something suitably weird and have new notation for the type?

Reply:
There are plenty of primitive type constructors -- references, raw pointers, tuples, arrays, slices -- they just all have special syntax as well instead of using angle brackets. I don't think special syntax for these new primitives is worth it.